tests/integration/questionnaire/test_questionnaire_list_collector_primary_person.py

```diff
@@ -68,3 +69,21 @@ def test_cannot_remove_primary_person_from_list_collector(self):
 
         self.assertInBody("James May")
         self.assertInBody("Marie Day")
+
+    def test_changing_answer_from_no_to_yes_on_primary_person_list_collector_resumes_in_right_location(
+        self,
+    ):
+        response_id = random.choices(string.digits, k=16)
+
+        # Given I initially answer 'No' to the primary person list collector
+        self.launchSurvey("test_list_collector_primary_person", reponse_id=response_id)
+        self.post({"you-live-here": "No"})
+
+        # When I change my answer to 'Yes' and sign out
+        self.get("questionnaire/primary-person-list-collector/")
+        self.post({"you-live-here": "Yes"})
+        self.post(action="sign_out")
```

Comment:
This should be `self.get("/sign-out")` now that the save on signout pr has merged.